.github/workflows/deploy.yml

```diff
@@ -0,0 +1,51 @@
+name: deploy
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+
+  deploy:
+    # this will run when you have tagged a commit, starting with "v*"
+    # and requires that you have put your twine API key in your
+    # github secrets (see readme for details)
+    if: ${{ github.repository == 'pyapp-kit/superqt' && contains(github.ref, 'tags') }}
```

Comment:
is the repo check intended?

Reply:
Whoops I forgot to update the repo.

Will fix. Also the failing check you mention seems unrelated to this PR. i believe the latest fixes for that are on #42?

Reply:
Can we have an integration branch with all the changes (working building with installation manager GUI) to ship to QA testing? We can worry about merging next week but I want to give QA the installation manager to test this week

Reply:
>(working building with installation manager GUI) 

I think I could, but I would still need to make some changes to the bundling part so that it includes all the needed packages but I would need @jaimergp advice 🤔 on that part.